docs/articles_en/learn_openvino/tutorials/notebooks_section_1.rst

```diff
@@ -113,12 +113,6 @@ Tutorials that explain how to optimize and quantize models with OpenVINO tools.
 
    Use Neural Network Compression Framework (NNCF) to quantize PyTorch model in post-training mode (without model fine-tuning).
 
-.. showcase::
```

Comment:
editing the notebook section needs to be done in alignment with the changes to notebooks themselves. let's keep these things for now